pkg/storaged/crypto/keyslots.jsx

```diff
@@ -253,9 +253,9 @@ function ensure_package_installed(steps, progress, package_name) {
                 text = _("Waiting for other software management operations to finish");
             } else if (p.package) {
                 let fmt;
-                if (p.info == PkEnum.INFO_DOWNLOADING)
+                if (p.info == InstallProgressType.DOWNLOADING)
                     fmt = _("Downloading $0");
-                else if (p.info == PkEnum.INFO_REMOVING)
+                else if (p.info == InstallProgressType.REMOVING)
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-22612-2d6fdce8-20251121-125419-fedora-43-devel/Coverage/lcov/github-pr.diff.gcov.html)